client/store/axiosErrorMiddleware.ts

```diff
@@ -1,14 +1,21 @@
 import axios, { AxiosStatic } from 'axios'
+import { Action, Dispatch, Middleware, MiddlewareAPI } from 'redux'
+import { v4 as uuidv4 } from 'uuid'
```

Comment:
dont we have UUIDs utils ?

Reply:
I don't think so. It seems there is no one.

Reply:
done.